.github/workflows/stable-cicd.yaml

```diff
@@ -82,8 +79,20 @@ jobs:
                     pypi_username: ${{secrets.PYPI_USERNAME}}
                     pypi_password: ${{secrets.PYPI_PASSWORD}}
                     ADMIN_GITHUB_TOKEN: ${{secrets.ADMIN_GITHUB_TOKEN}}
+            # At this point, the Roundup as published release/X.Y.Z to GitHub and PyPI and advanced
```

Comment:
@nutjob4life Is there any way we can grab the tag name `release/X.Y.Z` here, and use that to determine what version this should be? My only concern here is we are assuming this is a minor version increment, but what if it was a major or hotfix?

Reply:
Looks like we can: https://github.com/orgs/community/discussions/26686

Reply:
Ah sure, good point @jordanpadams. I pushed a new commit, 6e7631bb546e8a1c0f34583a822cd18bfed21d05, which simplifies and streamlines.